maven-plugin/src/main/java/io/confluent/kafka/schemaregistry/maven/derive/schema/DeriveSchema.java

```diff
@@ -162,12 +164,59 @@ public ObjectNode mergeArrays(List<JsonNode> arrayList, boolean useItems,
     return mergeMultipleDataTypes(mergedArray, primitives, records, arrays, check2dArray);
   }
 
+  protected JsonNode getNullSchema() {
+    return getPrimitiveSchema(mapper.convertValue(null, NullNode.class)).get();
+  }
+
+  protected ObjectNode getSchemaForMultipleMessages(List<JsonNode> messages)
```

Comment:
is this not required to be public like the one in DeriveJsonSchema ?

Reply:
Changed to public